modular_robot_physical/revolve2/modular_robot_physical/_physical_robot_config.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import pickle
+from dataclasses import dataclass
+
+from revolve2.modular_robot import ModularRobot
+from revolve2.modular_robot.body.base import ActiveHinge
+
+
+@dataclass
+class PhysicalRobotConfig:
+    """The configuration for running a physical robot."""
+
+    modular_robot: ModularRobot  # the modular robot object.
```

Comment:
```suggestion
    modular_robot: ModularRobot 
    """The modular robot object."""
```
Docstrings work for variables :)